src/commonmark/en/content/developer/web-api.md

```diff
@@ -9531,6 +9559,619 @@ dashboard item completely:
 
     /api/dashboards/<dashboard-id>/items/<item-id>/content/<content-resource-id>
 
+## Visualization
+
+<!--DHIS2-SECTION-ID:webapi_visualization-->
+
+The Visualization API is designed to help clients to interact with charts and pivot/report tables. The endpoints of this API are used by the Data Visualization application which allows the creation, configuration and management of charts and pivot tables based on the client's definitions. The main idea is to enable clients and users to have a unique and centralized API providing all types of charts and pivot tables as well as specific parameters and configuration for each type of visualization.
+
+This API was introduced with the expectation to unify both "charts" and "reportTables" APIs and entirely replace them in favour of the "visualizations" API (which means that the usage of "charts" and "reportTables" APIs should be avoided). In summary, the following resources/APIs:
+
+    /api/charts, /api/reportTables
+
+*are being replaced by*
+
+    /api/visualizations
+
+> **Note**
+>
+> New applications and clients should avoid using the "charts" and "reportTables" APIs because they are deprecated. Use the "visualizations" API instead.
```

Comment:
Use backticks instead of double quote for API resources, e.g. `charts`.

Reply:
Done